api/rs/slint/mcu.md

```diff
@@ -387,9 +388,9 @@ loop {
 ```
 
 Note: In our experience, using the synchronous `DrawTarget::fill_contiguous` function is slow. If
-your device is capable of using DMA, you may be able to achieve better performance by using
-two line buffers: One buffer to render into with the CPU, while the other buffer is transferred to
-the screen using DMA asynchronously.
+your device is capable of DMA, you can improve performance by switching between
+two line buffers. Use one buffer to render into using the CPU. The other buffer
+can transfer data to the screen using DMA in the meantime.
```

Comment:
This isn't correct I think. The buffer is not the entity that transfers data to the screen. It is the buffer that *is* transferred (passive) using DMA. I'd also keep the asynchronously part since that is crucial to the advertised performance improvement.